src/SFA.DAS.LevyTransferMatching.Api/Controllers/OpportunityController.cs

```diff
@@ -272,5 +273,26 @@ public async Task<IActionResult> Detail(int opportunityId)
                 return new StatusCodeResult((int)HttpStatusCode.InternalServerError);
             }
         }
+
+        [HttpGet]
+        [Route("opportunities/{opportunityId}/apply")]
+        [ProducesResponseType((int)HttpStatusCode.OK)]
+        public async Task<IActionResult> OpportunityApply(int opportunityId, string userId)
```

Comment:
See corresponding points about the naming of this in the UI PR.

Reply:
The corresponding APIM classes have also been renamed.